### PR TITLE
Optional arguments in mchirp, q <-> mass1, mass2 transform

### PR DESCRIPTION
**Pull request by cdcapano**
This allows the name of the parameters used for mass1, mass2, mchirp, and q for the `MchirpQtoMass1Mass2` transform (and inverse) to be an option set in the configuration file. This is needed to sample in source masses instead of detector frame masses (in which case the prior parameter is something like `srcmass(1|2)` instead of `mass(1|2)`).

Default is to use the standard parameter names, so no changes need to be made to the standard config files.

### CODE REVIEW

File: pycbc/transforms.py [hunk 1]
```diff
@@ -1492,13 +1509,30 @@ class Mass1Mass2ToMchirpQ(MchirpQToMass1Mass2):
...
+            mchirp_param = parameters.mchirp
+        if q_param is None:
+            q_param = parameters.q
+        self.mass1_param = mass1_param
+        self.mass2_param = mass2_param
+        self.mchirp_param = mchirp_param
+        self.q_param = q_param
+        self._inputs = [self.mass1_param, self.mass2_param]
+        self._outputs = [self.mchirp_param, self.q_param]
+        super(MchirpQToMass1Mass2, self).__init__()
```
Comment (cmbiwer) [inline pycbc/transforms.py]: I think codacy is right that this should be `` super(Mass1Mass2ToMchirpQ, self).__init__()``.

Comment (cdcapano) [reply pycbc/transforms.py]: @cmbiwer Ok, I called the `BaseTransform` directly there.

**Comment by cdcapano** [comment]
@cmbiwer Catching up on old PRs... I replaced the call to `super` with a direct call to the appropriate class, as requested.

**Comment by cdcapano** [comment]
@cmbiwer poke

**Comment by cdcapano** [comment]
Making @dfinstad a reviewer since @cmbiwer hasn't responded.

**Comment by cdcapano** [comment]
Thanks @dfinstad... looks like I can't commit until @cmbiwer approves since he requested changes.

**Comment by dfinstad** [comment]
Should be good to go now @cdcapano

**Pull request merged**